<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>openBook</class>
 <widget class="QDialog" name="openBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>451</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_BookTitle">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <italic>false</italic>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Title:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_BookAuthor">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Author: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_BookYear">
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Pubuish Year: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_CurrentStatus">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>current Status:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser_discrpsion"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_BookFav">
        <property name="text">
         <string>Favourit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_BookCheckOutReserve">
        <property name="text">
         <string>Check Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>openBook</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>380</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
